subsys/usb/class/netusb/function_rndis.c

```diff
@@ -409,33 +413,12 @@ static void rndis_bulk_out(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
 			return;
 		}
 
-		buf = net_pkt_get_frag(pkt, K_NO_WAIT);
-		if (!buf) {
-			/* In a case of low memory skip the whole packet
-			 * hoping to get buffers for later ones
-			 */
-			rndis.skip_bytes = rndis.in_pkt_len - len;
-
-			LOG_ERR("Not enough net buffers, len %u, skip %u",
-				rndis.in_pkt_len, rndis.skip_bytes);
-
-			net_pkt_unref(pkt);
-			rndis.rx_no_buf++;
-			return;
-		}
-
-		net_pkt_frag_insert(pkt, buf);
-
 		rndis.in_pkt = pkt;
-
-		/* Append data only, skipping RNDIS header */
-		hdr_offset = sizeof(struct rndis_payload_packet);
 	}
 
-	ret = net_pkt_append_all(rndis.in_pkt, len - hdr_offset,
```

Comment:
This removes the use of ret, so the corresponding variable declaration needs to be removed from line 341, otherwise it causes a warning.